app/models/fcrepo_endpoint.rb

```diff
@@ -12,11 +12,15 @@ def self.reset!
   end
 
   def ping
-    ActiveFedora::Fedora.instance.connection.head(
+    if ActiveFedora::Fedora.instance.connection.head(
       ActiveFedora::Fedora.instance.connection.connection.http.url_prefix.to_s
     ).response.success?
-  rescue StandardError
-    false
+      "Fedora is OK"
+    else
+      "Fedora is Down"
```

Comment:
I'm wondering about unifying the messages, which might make monitoring easier.

```suggestion
      "Error checking Fedora status: Fedora is Down"
```

Reply:
@jeremyf Ahhh! Thank you! Love it! I appreciate the refactor here!